### PR TITLE
[SPARK-28574][CORE][FOLLOW-UP] Several minor improvements for event queue capacity config

### PR DESCRIPTION
**Pull request by Ngone51**
### What changes were proposed in this pull request?

* Replace hard-coded conf `spark.scheduler.listenerbus.eventqueue` with a constant variable(`LISTENER_BUS_EVENT_QUEUE_PREFIX `) defined in `config/package.scala`.

* Update documentation for `spark.scheduler.listenerbus.eventqueue.capacity` in both `config/package.scala` and `docs/configuration.md`.

### Why are the changes needed?

* Better code maintainability

* Better user guidance of the conf

### Does this PR introduce any user-facing change?

No behavior changes but user will see the updated document.

### How was this patch tested?

Pass Jenkins.

### CODE REVIEW

**Comment by Ngone51** [comment]
cc @gatorsmile @jiangxb1987 @cloud-fan @yunzoud

File: core/src/main/scala/org/apache/spark/internal/config/package.scala [hunk 1]
```diff
@@ -596,6 +597,11 @@ package object config {
 
   private[spark] val LISTENER_BUS_EVENT_QUEUE_CAPACITY =
     ConfigBuilder("spark.scheduler.listenerbus.eventqueue.capacity")
+      .doc("The capacity config for listener bus event queue when the event queue doesn't " +
```
Comment (cloud-fan) [inline core/src/main/scala/org/apache/spark/internal/config/package.scala]: nit:
```
The default capacity for event queues. Spark will try to initialize an event queue using capacity specified
by `spark.scheduler.listenerbus.eventqueue.queueName.capacity` first. If it's not configured, Spark will
use the default capacity specified by this config.
```

**Review by cloud-fan — APPROVED** [review]

**Review by dongjoon-hyun — APPROVED** [review]
+1, LGTM. Merged to master.
Thank you, @Ngone51 , @cloud-fan , @HyukjinKwon .

File: docs/configuration.md [hunk 2]
```diff
@@ -1852,9 +1852,11 @@ Apart from these, the following properties are also available, and may be useful
   <td><code>spark.scheduler.listenerbus.eventqueue.capacity</code></td>
   <td>10000</td>
   <td>
-    Capacity for event queue in Spark listener bus, must be greater than 0. Consider increasing
-    value (e.g. 20000) if listener events are dropped. Increasing this value may result in the
-    driver using more memory.
+    The default capacity for event queues. Spark will try to initialize an event queue 
+    using capacity specified by `spark.scheduler.listenerbus.eventqueue.queueName.capacity`
```
Comment (jiangxb1987) [inline docs/configuration.md]: `spark.scheduler.listenerbus.eventqueue.queueName.capacity` -> `spark.scheduler.listenerbus.eventqueue.${queueName}.capacity`

**Comment by Ngone51** [comment]
Thanks @dongjoon-hyun  @cloud-fan  @HyukjinKwon .

File: docs/configuration.md (hunk 2 above)
Comment (Ngone51) [reply docs/configuration.md]: err... shall we do another follow up ?

**Pull request closed without merging**